http/org.wso2.carbon.transport.http.netty/src/main/java/org/wso2/carbon/transport/http/netty/listener/WebSocketSourceHandler.java

```diff
@@ -200,6 +233,8 @@ private void setupCarbonMessage(ChannelHandlerContext ctx) {
         cMsg.setProperty(Constants.REMOTE_PORT, ((InetSocketAddress) ctx.channel().remoteAddress()).getPort());
```

Comment:
We need to remove these REMOTE stuff, specially this line ((InetSocketAddress) ctx.channel().remoteAddress()).getHostName()